File: app/src/main/java/com/thorvg/sample/MainActivity.java
package com.thorvg.sample;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.thorvg.android.widget.LottieAnimationView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        LottieAnimationView lottieView = findViewById(R.id.lottieView);
        lottieView.setLottieDrawable(R.drawable.lottie_swinging);
    }
}

File: sample/src/main/java/com/thorvg/sample/MainActivity.kt
package com.thorvg.sample

import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.thorvg.android.widget.LottieAnimationView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val lottieView = findViewById<LottieAnimationView>(R.id.lottie_view)
        lottieView.setLottieDrawable(R.drawable.lottie_swinging)

        findViewById<View>(R.id.anim_state).setOnClickListener { v: View ->
            val button = v as TextView
            if ("Pause".contentEquals(button.text)) {
                lottieView.pauseAnimation()
                button.text = "Resume"
            } else {
                lottieView.resumeAnimation()
                button.text = "Pause"
            }
        }
    }
}